test/Airgrab.test.ts

```diff
@@ -82,6 +85,11 @@ describe('Airgrab', function () {
       provider as any,
     );
 
+    // if airdrop is finished skip tests
+    if ((await airgrab.endTimestamp()) < (await helpers.time.latest())) {
+      this.skip();
+    }
```

Comment:
I think at this point that the airgrab already finished on mainnet we can probably just skip the whole file all together (so having a skip at the top level). Or do you see any value in these?